src/XrdSec/XrdSecEntity.hh

```diff
@@ -69,13 +69,29 @@ const    char   *tident;                  // Trace identifier always preset
                                           // it in your XrdSfsFileSystem::Disc()
                                           // implementation, as needed.
          XrdSecEntity(const char *pName = "")
-                     : name(0), host(0), vorg(0), role(0), grps(0),
-                       endorsements(0), moninfo(0), creds(0), credslen(0),
-                       rsvd(0), addrInfo(0), tident(""), sessvar(0)
-                     {strncpy(prot, pName, XrdSecPROTOIDSIZE-1);
+                     {Reset();
+	              strncpy(prot, pName, XrdSecPROTOIDSIZE-1);
                       prot[XrdSecPROTOIDSIZE-1] = '\0';
```

Comment:
How does this square with ABI compatability? The in-lining differs.

Reply:
The code compiled against the old version of constructor will have the old version of the constructor inlined, there's no ABI compatibility problem. The new and the old version of the constructor are functionally equivalent so it is OK.

Reply:
Well, that's only true if the Reset() is actually inlined, isn't it?But I guess we are not providing regressed compatibility (i.e. new code running with old libraries).